frontend/pages/signup.tsx

```diff
@@ -149,31 +156,45 @@ export default function Signup() {
   };
 
   return (
-    <Layout
-      title={texts.sign_up}
-      isLoading={isLoading}
-      message={errorMessage}
-      messageType={errorMessage && "error"}
-    >
-      {curStep === "basicinfo" ? (
-        <BasicInfo
-          values={userInfo}
-          handleSubmit={handleBasicInfoSubmit}
-          errorMessage={errorMessages[steps[0]]}
-        />
-      ) : (
-        curStep === "personalinfo" && (
-          <AddInfo
-            values={userInfo}
-            handleSubmit={handleAddInfoSubmit}
-            errorMessage={errorMessages[steps[1]]}
-            handleGoBack={handleGoBackFromAddInfo}
-            locationInputRef={locationInputRef}
-            locationOptionsOpen={locationOptionsOpen}
-            handleSetLocationOptionsOpen={handleSetLocationOptionsOpen}
-          />
-        )
-      )}
-    </Layout>
+    <WideLayout message={errorMessage} messageType={errorMessage && "error"} isLoading={isLoading}>
```

Comment:
I think the `WideLayout` component needs a title prop to set the title for the title tag.

Reply:
I’ve finished the review, and the code looks great!